src/SoftwareLicense.php

```diff
@@ -1258,45 +1185,40 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
                     return self::createTabEntry(
                         self::getTypeName(Session::getPluralNumber()),
                         (($nb >= 0) ? $nb : '&infin;'),
-                        $item::getType()
+                        $item::class
                     );
 
-                case SoftwareLicense::class:
+                case __CLASS__:
```

Comment:
We use the `::class` notation everywhre. It seems more logical to use `self::class` here.
```suggestion
                case self::class:
```